Utils/CameraControl27.py

```diff
@@ -1,8 +1,14 @@
 """ Controls ONVIF-compatible IP camera """
 
+# Prerequisites:
+# Python2 
+#   pip install onvif
```

Comment:
Maybe it's a cool idea add these packages into `setup.py` and `requirements.txt` files.

Reply:
Actually, its there already. It was added some time ago. I just clarified the comments in the module header. This is from lines 30 onwards in Setup.py.
------
### HANDLE DIFFERENT ONVIF LIBRARIES FOR Py 2 AND 3 ###

# Python 2 uses the 'onvif' library and Python 3 uses the onvif_zeep library
if sys.version_info[0] < 3:

    onvif_str = 'onvif_zeep'
    onvif_proper = 'onvif'

else:
    onvif_str = 'onvif'
    onvif_proper = 'onvif_zeep'